### DISCUSSION TITLE
while podman pull, got lsetxattr /bin: operation not supported

### DISCUSSION THREAD

**Discussion by PrincePranav235** [containers/podman#24357] [General]
while pulling an image using i got this error , someone help me with this.
`podman pull ghcr.io/aquasecurity/trivy:latest
WARN[0000] Network file system detected as backing store.  Enforcing overlay option `force_mask="700"`.  Add it to storage.conf to silence this warning
Trying to pull ghcr.io/aquasecurity/trivy:latest...
Getting image source signatures
Copying blob 4ff82ebd20fe done   |
Copying blob 43c4264eed91 done   |
Copying blob 2fb915365b73 done   |
Copying blob 55dd28896ea8 done   |
Error: copying system image from manifest list: writing blob: adding layer with blob "sha256:43c4264eed91be63b206e17d93e75256a6097070ce643c5e8f0379998b44f170": processing tar file(lsetxattr /bin: operation not supported): exit status 1
`

**Comment by PrincePranav235** [comment]
why the above operation not supported issue is there,
how can i resolve this in my system,
for your info, i am on linux system

**Comment by rhatdan** [comment]
Please provide podman info.  Is your homedir on an NFS or network storage?  If yes then this is likely not going to work, since the remote storage server will not allow users to create UIDs other then the primary UID.

**Reply by PrincePranav235** [reply]
@rhatdan 
yes it is a network storage.
So, what is the resolution to this?

**Reply by PrincePranav235** [reply]
graphDriverName: overlay
  graphStatus:
    Backing Filesystem: nfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Supports shifting: "true"
    Supports volatile: "true"
    Using metacopy: "false"
  imageCopyTmpDir: /var/tmp
  imageStore:
    number: 0
  runRoot: /run/user/1897876/containers
  transientStore: false
  volumePath: /home/prpranav/.local/share/containers/storage/volumes
version:
  APIVersion: 4.9.4-rhel
  Built: 1727192768
  BuiltTime: Tue Sep 24 15:46:08 2024
  GitCommit: ""
  GoVersion: go1.21.13 (Red Hat 1.21.13-2.module+el8.10.0+90407+20770c94)
  Os: linux
  OsArch: linux/amd64
  Version: 4.9.4-rhel
